plugins/inputs/cisco_telemetry_mdt/cisco_telemetry_mdt.go

```diff
@@ -391,9 +1394,72 @@ func (c *CiscoTelemetryMDT) parseKeyField(tags map[string]string, field *telemet
 	}
 }
 
+func (c *CiscoTelemetryMDT) parseRib(grouper *metric.SeriesGrouper, field *telemetry.TelemetryField, prefix string, path string, tags map[string]string, timestamp time.Time) {
+	// RIB
+	measurement := path
+	for _, subfield := range field.Fields {
+		//For Every table fill the keys which are vrfName, address and masklen
+		switch subfield.Name {
+		case "vrfName", "address", "maskLen":
+			tags[subfield.Name] = decodeTag(subfield)
+		}
+		if value := decodeValue(subfield); value != nil {
+			grouper.Add(measurement, tags, timestamp, subfield.Name, value)
+		}
+		if subfield.Name == "nextHop" {
```

Comment:
Invert the logic here to save some indention level(s).
```suggestion
		if subfield.Name != "nextHop" {
			continue
		}
```